test/TestSupport.js

```diff
@@ -0,0 +1,22 @@
+
+module.exports = { 
+    buildSimpleTimelineRangeBody : function() {
+        var body = {
+            dom: {
+                center: {
+                clientWidth: 1000
+                }
+            },
+            domProps: this.props,
+            emitter: {
+                on: function() {},
+                off: function() {},
+                emit: function() {}
+            },
+            hiddenDates: [],
+            util: {}
+            };
+            body.dom.rollingModeBtn = document.createElement('div');

```

Comment:
I didn't (well, not in this PR). That is existing code from https://github.com/almende/vis/blob/develop/test/TimelineRange.test.js#L24 that is needed for the part of the Range constructor that adds an event listener: https://github.com/almende/vis/blob/develop/lib/timeline/Range.js#L69